jdaviz/core/loaders/resolvers/url/url.py

```diff
@@ -38,6 +40,13 @@ def is_valid(self):
 
     @observe('url', 'cache', 'timeout')
     def _on_url_changed(self, change):
+        # _uri_output_file is a string to the local path which uses the same file name
+        # as that from the URL. By stripping the local path, we can match to the URL.
+        # We check for bool for 'cache' on/off.
+        if (self._uri_output_file.lstrip(f"{self.local_path}") not in self.url.strip()
+                or isinstance(change['new'], bool)):
+            del self._uri_output_file
```

Comment:
can we add a comment saying that this clears the cache and will force a re-download (or read from the local cache)?